src/java/picard/analysis/CollectWgsMetricsFromQuerySorted.java

```diff
@@ -32,20 +32,31 @@
     @Option(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Output metrics file.")
     public File OUTPUT;
 
-    @Option(shortName = "MQ", doc = "Minimum mapping quality for a read to contribute coverage.", overridable = true)
-    public int MINIMUM_MAPPING_QUALITY = 20;
+    @Option(shortName = "USABLE_MQ", doc = "Minimum mapping quality for a read to contribute to usable coverage.", overridable = true)
+    public int MINIMUM_USABLE_MAPPING_QUALITY = 20;
 
-    @Option(shortName = "Q", doc = "Minimum base quality for a base to contribute coverage.", overridable = true)
-    public int MINIMUM_BASE_QUALITY = 20;
+    @Option(shortName = "USABLE_Q", doc = "Minimum base quality for a base to contribute to usable coverage.", overridable = true)
+    public int MINIMUM_USABLE_BASE_QUALITY = 20;
+
+    @Option(shortName = "RAW_MQ", doc = "Minimum mapping quality for a read to contribute to raw coverage.", overridable = true)
+    public int MINIMUM_RAW_MAPPING_QUALITY = 0;
+
+    @Option(shortName = "RAW_Q", doc = "Minimum base quality for a base to contribute to raw coverage.", overridable = true)
+    public int MINIMUM_RAW_BASE_QUALITY = 3;
 
     private final Log log = Log.getInstance(CollectWgsMetricsFromQuerySorted.class);
 
+    public enum METRICS_TYPE { RAW, USABLE }
+
     /** Metrics for evaluating the performance of whole genome sequencing experiments. */
     public static class QuerySortedSeqMetrics extends CollectWgsMetrics.WgsMetrics {
+        /** Identifier for metrics type */
+        public METRICS_TYPE TYPE;
```

Comment:
I suppose looking at the functionality closer, you are adding duplicate option pairs here so you can do one-pass but compute the same metrics with different thresholds.  So without having to support an arbitrary number of cutoff tuples, perhaps rename `RAW` and `USABLE` to `LOW` and `HIGH`, or `LENIENT` and `STRINGENT`?  Raw and usable don't make a lot of sense to me without some internal Broad context.  In the latter case, type is `STRINCENCY_TYPE`?  I will leave this up to you folks to decide, as I think expedience of implementation may be trumping how these options are presented to external users.